packages/AreaChart/src/AreaChart.tsx

```diff
@@ -387,14 +384,26 @@ const AreaChart: React.FunctionComponent<AreaChartProps> = ({
         stroke: "none",
         fill: "none"
     };
-
     const buildAreaDefs = (): JSX.Element => {
         return (
             <defs>
-                <linearGradient id="curveAreaFillGradient" x1="0" y1="0" x2="0" y2="1">
-                    <stop stopColor={variables.electricBlue50} />
-                    <stop offset="78%" stopColor="#EDF6FF" />
-                    <stop offset="96%" stopColor={variables.samoyed} />
+                <linearGradient 
+                    id="lineGradient" 
+                    x1="0" 
+                    y1="0" 
+                    x2="0"
+                    y2="96%"
+                    gradientUnits="userSpaceOnUse"
+                >
+                    <stop stopColor="#188A71" />
+                    <stop offset="0.119792" stopColor="#47A584" />
```

Comment:
Are these the colors that should be in the dataviz tokens?

Reply:
yes

Reply:
I've seen that you've created an issue for this in Hopper, perhaps you should create one in igloo to replace the set of values with the new library when it becomes available.